examples/inverse/evoked_ers_source_power.py

```diff
@@ -153,3 +163,66 @@ def _gen_mne(active_cov, baseline_cov, common_cov, fwd, info, method='dSPM'):
 brain_dspm = stc_dspm.plot(
     hemi='rh', subjects_dir=subjects_dir, subject=subject,
     time_label='dSPM source power in the 12-30 Hz frequency band')
+
+# %%
+# Use volume, time-frequency source estimates
+# -------------------------------------------
+
+# make a volume source space
+surface = subjects_dir / subject / 'bem' / 'inner_skull.surf'
+vol_src = mne.setup_volume_source_space(
+    subject=subject, subjects_dir=subjects_dir, surface=surface,
+    pos=10, add_interpolator=False)  # just for speed!
+
+conductivity = (0.3,)  # one layer for MEG
+model = mne.make_bem_model(subject=subject, ico=3,  # just for speed
+                           conductivity=conductivity,
+                           subjects_dir=subjects_dir)
+bem = mne.make_bem_solution(model)
+
+trans = fwd['info']['mri_head_t']
+vol_fwd = mne.make_forward_solution(
+    raw.info, trans=trans, src=vol_src, bem=bem, meg=True, eeg=True,
+    mindist=5.0, n_jobs=1, verbose=True)
+
+# Compute source estimate, just use MNE
+snr = 3.0
+lambda2 = 1.0 / snr ** 2
+method = 'MNE'  # use dSPM method (could also be dSPM or sLORETA)
+
+# make a different inverse operator for each frequency so as to properly
+# whiten
+inverse_operator = list()
+for freq_idx in range(epochs_tfr.freqs.size):
+    baseline_cov = baseline_csd.get_data(index=freq_idx, as_cov=True)
+    baseline_cov['data'] = baseline_cov['data'].real  # only normalize by real
+    inverse_operator.append(mne.minimum_norm.make_inverse_operator(
+        epochs.info, vol_fwd, baseline_cov))
+
+stcs = mne.minimum_norm.apply_inverse_tfr_epochs(
+    epochs_tfr, inverse_operator, lambda2, method=method,
+    pick_ori='vector')
+
+# %%
+# Plot volume source estimates
+# ----------------------------
+
+# note, here frequencies are the outer list, opposite of the beamformer
+# here, we used pick_ori='vector' so we have an orientation dimension
+
+# compute power, take the average over epochs and cast to integers to save
+# memory, the GUI can also handle complex data across epochs if your
+# computer has enough RAM but this really lowers the memory usage
+data = np.array([(np.mean(
+    [(stc.data * stc.data.conj()).real for stc in tfr_stcs],
+    axis=0, keepdims=True) * 1e32).astype(np.uint64) for tfr_stcs in stcs])
+data = data.transpose((1, 2, 3, 0, 4))  # move frequencies to penultimate
+
+# gain normalize
+data = data // data.mean(axis=-1, keepdims=True)
+
+viewer = mne.gui.view_stc(data, subject=subject, subjects_dir=subjects_dir,
```

Comment:
I'm happy to change the name but it already supports time data only and has tests for it so I think adding TFR to the name doesn't make sense. This is the same conversation as @larsoner and I had for `SourceEstimateViewer` -> `VolSourceEstimateViewer`.

The array is not ideal but we need a new time-frequency source estimate class to handle this data type. I'm a strong -1 for having the list of lists because of the complexity of describing the parameter. It also makes it incompatible for the beamformer since the lists are in opposite order.

Reply:
> I'm happy to change the name

yes, please do.

> but it already supports time data only so I think adding TFR to the name doesn't make sense.

it's OK to have things be *more functional* than they're documented to be, esp. when under heavy development.

> The array is not ideal but we need a new time-frequency source estimate class to handle this data type.

Yep, we do, as we've discussed many times. When that happens, we'll make whatever changes are necessary here.  In the meantime, list-or-generator-of-stcs is what several of us have suggested / requested, multiple times.

Reply:
The name was changed to `view_vol_stc` to match.